server/routes/notification.js

```diff
@@ -0,0 +1,17 @@
+const express = require("express");
+const router = express.Router();
+const protect = require("../middleware/auth");
+
+const {
+  retrieveUsersNotifications,
+  retrieveUsersUnreadNotifications,
+  createNotification,
+  markAsRead,
+} = require("../controllers/notification");
+
+router.route("/").post(protect, createNotification);
+router.route("/all").get(protect, retrieveUsersNotifications);
+router.route("/unread").get(protect, retrieveUsersUnreadNotifications);
+router.route("/mark-read/:notificationId").patch(protect, markAsRead);
```

Comment:
Looks like it's just a documentation error above, you made it a `PATCH` request here 👍🏼 

Also, just to follow REST naming conventions, it would make more to swap these around: `notifications/:notificationId/mark-read`. Although, these aren't strict rules, I'd still recommend skimming through this [article](https://restfulapi.net/resource-naming/) to learn more about REST api naming conventions.

Reply:
Ok ill change this and make a note to change this on API requests on the FE as well.